sonar-helm-for-iac/src/org.sonar.iac.helm/template-evaluation.pb.go

```diff
@@ -18,7 +18,7 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.31.0
+// 	protoc-gen-go v1.33.0
```

Comment:
Why is this committed? Locally I have the same change.
I guess some software update updated also `protoc-gen-go` on our machines. 
Maybe it should be updated in separate PR + CI configuration?

Reply:
Yes, `protoc-gen-go` was updated by Renovate (with manual additions by me), but I must've not regenerated the code. Yes, I can look at it as a separate thing and rollback here. Probably, something like a simple check in the build task to ensure that no changes are left uncommitted in the Go code should do.

Reply:
https://github.com/SonarSource/sonar-iac/pull/1289